plugin/mouse/mouse_scroll.py

```diff
@@ -1,20 +1,22 @@
 import time
-from typing import Literal
+from typing import Literal, Optional
 
 from talon import Context, Module, actions, app, cron, ctrl, imgui, settings, ui
 
+DEFAULT_CONTINUOUS_SCROLLING_SPEED_FACTOR: int = 10
```

Comment:
I'm inclined to make this a setting. Any reason not to do so?

Reply:
I can make that change.

Reply:
I made it a setting.